frontend/tests/ui/test_contest.py

```diff
@@ -281,7 +281,8 @@ def create_contest_admin(driver, contest_alias, problem, users, user,
             driver.wait.until(
                 EC.element_to_be_clickable(
                     (By.XPATH,
-                     '//a[starts-with(@href, "%s")]' % contest_url))).click()
+                     '//small/a[contains(@href, "%s")]' % contest_url
```

Comment:
```suggestion
                     '//small/a[starts-with(@href, "%s")]' % contest_url
```

esto estaba bien con `starts-with`.

Reply:
Listo, se regresa al `starts-with`
`